torch-neuronx/transformers-neuronx/inference/CodeLlama-7B.ipynb

```diff
@@ -0,0 +1,278 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "59913016-f89e-4a0e-9afe-b3a06e9112d5",
+   "metadata": {},
+   "source": [
+    "# Run `codellama/CodeLlama-7b-hf` from Hugging Face on Inf2 & Trn1"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f8454655-ec27-45e3-8da7-f82b744321ee",
+   "metadata": {},
+   "source": [
+    "In this example we compile and deploy the [codellama/CodeLlama-7b-hf](https://huggingface.co/codellama/CodeLlama-7b-hf) model from Hugging Face on Neuron core devices using the `transformers-neuronx` package.\n",
+    "\n",
+    "The example has the following main sections:\n",
+    "1. Deploy your instance\n",
+    "1. Set up the Jupyter Notebook\n",
+    "1. Inference from a pre-compiled model\n",
+    "1. Compiling a model\n",
+    "1. Save to disk\n",
+    "\n",
+    "\n",
+    "This Jupyter Notebook can be run on a Inf2 instance (`inf2.xlarge`) or larger or also a Trn1 instance (`trn1.32xlarge`)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a649c112",
+   "metadata": {},
+   "source": [
+    "## Deploy your instance\n",
+    "\n",
+    "1. Raise your limits\n",
+    "    The default limit for Inferentia and Trainium instances are 0, so request an increase before you start!  EC2 instances are limited by vCPUs, but SageMaker limits are based on number of instances and instance size.  Start with a vCPU limit request of 32 so you can deploy an inf2.xlarge (4 vCPUs) or an inf2.8xlarge (32 vCPUs).\n",
```

Comment:
Provide a link to allow users to submit requests for limit raises.

Reply:
Added.  The console links are region specific for Service Quotas, but I think it is OK to lead people to us-west-2 and it says later that quotas are region specific.